networkx/algorithms/traversal/breadth_first_search.py

```diff
@@ -152,19 +173,28 @@ def bfs_tree(G, source, reverse=False):
     >>> G = nx.path_graph(3)
     >>> print(list(nx.bfs_tree(G,1).edges()))
     [(1, 0), (1, 2)]
+    >>> H = nx.Graph()
+    >>> nx.add_path(G, [0, 1, 2, 3, 4, 5, 6])
+    >>> nx.add_path(G, [2, 7, 8, 9, 10])
+    >>> dict(nx.bfs_tree(H, source=3, depth_limit=3)
```

Comment:
This tests that the depth_limit is 3, but the answer only shows a depth_limit of 1. (1 step away from the source.)